src/backend/MoveList.js

```diff
@@ -1,16 +1,66 @@
-import React from 'react';
+import React, {useContext, useEffect, useState} from 'react';
 import { Button } from 'react-bootstrap';
 import {all_moves, responses, updateTurnInDB} from './move_logic.js';
-
+import {firestore, root} from "../config/firebase";
+import firebase from 'firebase';
 
 
 export function MoveList(props){
+	const [assassinEnabled, setAssassin] = useState(true);
+	const [coupEnabled, setCoup] = useState(true);
+	const [theRest, setTheRest] = useState(false);
+
+	useEffect( () => {
+		const subscribe = firestore.collection(root).doc(props.roomName).collection("players").doc(props.activePlayerID).get().then((player)=>{
+			let coins = player.data().coins;
+			if (coins >= 3 && coins < 10){
+				setAssassin(false);
+			} 
+			if (coins >= 7){
+				setCoup(false);
```

Comment:
this function name and variable it sets is somewhat misleading. Can you rename to disableCoup so that it makes to pass false here